fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/VersionIndexTest.java

```diff
@@ -269,18 +388,18 @@ public void saveLoadWithVersion(int testFormatVersion, boolean testSplitLongReco
 
         try (FDBRecordContext context = openContext(simpleVersionHook)) {
             FDBStoredRecord<Message> stored1 = recordStore.loadRecord(Tuple.from(1066L));
-            assertThat(stored1.hasVersion(), is(true));
+            assertTrue(stored1.hasVersion());
```

Comment:
When this was first written, the error message for `assertTrue` and `assertFalse` weren't very good (they just said "assertion error"). Now, they say, `expected <true> but was <false>` (and vice versa), so we should prefer them in cases like this. I've updated this file in this PR, but it isn't totally related other than that I was editing this file.